test/bindings-colon-test.js

```diff
@@ -932,4 +933,15 @@ QUnit.test("on:input:value:to does not initialize values (#289)", function() {
 	}
 });
 
+QUnit.test("errors subproperties of undefined properties (#298)", function() {
+	var teardown = testHelpers.dev.willError(/Attempting to set a value at (.+) where (.+) is undefined./);
+	var template = stache("<input value:to='prop.subprop'/>");
+
+	var ViewModel = DefineMap.extend({});
+	var vm = new ViewModel();
+
+	template(vm);
+	QUnit.equal(teardown(), 1, "saw errors");
```

Comment:
Maybe these should just test that there isn't an error? If we change the warning in `can-view-scope`, we don't really need these tests to break.